client/my-sites/sidebar-unified/fallback-data.js

```diff
@@ -0,0 +1,523 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+const shouldShowLinks = true;
+const shouldShowTestimonials = true;
+const shouldShowPortfolio = true;
+const shouldShowWooCommerce = true;
+const shouldShowApperanceHeaderAndBackground = true;
+const shouldShowAdControl = true;
+const shouldShowAMP = true;
+const showShowThemeOptions = true;
```

Comment:
In future I expect that these options will need to be passed _into_ the function as args with defaults specified. This will make the function more testable and avoid building into too many dependencies.